corehq/apps/app_manager/tests/test_suite_training_module.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.test import SimpleTestCase
+
+from corehq.apps.app_manager.models import Application, TrainingModule
+from corehq.apps.app_manager.tests.util import TestXmlMixin
+
+
+class TrainingModuleSuiteTest(SimpleTestCase, TestXmlMixin):
+
+    def test_training_module(self):
```

Comment:
It also would be good to think about tests for training modules as shadow modules and as parent modules, if those are going to be supported. They sound like edge cases, but the menu-generating code is a rich source of bugs, especially where different kinds of modules interact with each other.

Reply:
Updated the PR so training modules cannot be parents or children.